Mac/BuildScript/build-installer.py

```diff
@@ -190,6 +196,27 @@ def getTargetCompilers():
 def internalTk():
     return getDeptargetTuple() >= (10, 6)
 
+
+def tweak_tcl_build(basedir, archList):
```

Comment:
@ned-deily  I'm not too happy about this, for some reason I needed this to get a working build.

Reply:
@ronaldoussoren build-installer's Tcl/Tk build steps need to be tweaked for building on 10.15+. 8.6.8 built on 10.9 executes reasonably well on newer systems but 8.6.8 build on, say, 10.15 doesn't.  I'll hack on that after we get the rest in.